nuxeo-services/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -59,13 +63,16 @@
 import org.nuxeo.ecm.core.schema.types.primitives.StringType;
 import org.nuxeo.ecm.platform.filemanager.api.FileImporterContext;
 
+import static org.nuxeo.ecm.core.api.LifeCycleConstants.INITIAL_LIFECYCLE_STATE_OPTION_NAME;
+import static org.nuxeo.ecm.core.query.sql.NXQL.ECM_LIFECYCLESTATE;
```

Comment:
It's a detail but our standard formatter puts static imports before regular ones.

Reply:
updated